content/docs/1.10.0/nodes-and-volumes/volumes/create-volumes.md

```diff
@@ -132,16 +131,18 @@ Only detached volume can be used by a newly created pod.
 ### The Failure of the Longhorn Volume Creation
 
 Creating a Longhorn volume will fail if there are no available nodes, disks, or insufficient storage. The failures are categorized into:
-- insufficient storage,
-- disk not found,
-- disks are unavailable,
-- failed to retrieve scheduling settings failed to retrieve,
-- tags not fulfilled,
-- node not found,
-- nodes are unavailable,
-- none of the node candidates contains a ready engine image,
-- hard affinity cannot be satisfied,
-- replica scheduling failed.
+
+- disk not found
+- disks are unavailable
+- failed to retrieve scheduling settings failed to retrieve
+- tags not fulfilled
+- node not found
+- nodes are unavailable
+- none of the node candidates contains a ready engine image
+- hard affinity cannot be satisfied"
+- replica scheduling failed
+- precheck new replica failed
+- evict replica failed
```

Comment:
What's the precheck failure for the new replica? 
Why does failing to evict the replica relate to volume creation failure?

Reply:
https://github.com/longhorn/longhorn-manager/pull/3989